packages/components/src/theme/test/color-algorithms.ts

```diff
@@ -53,9 +53,9 @@ describe( 'Theme color algorithms', () => {
 				'wp.components.Theme: The background color ("#000") does not have sufficient contrast against the accent color ("#111").'
 			);
 
-			generateThemeVariables( { background: '#eee' } );
+			generateThemeVariables( { background: '#1a1a1a' } );
```

Comment:
Changing this to a color that does trigger a contrast error with the new accent color.

Reply:
Can you expand on this? We still want to use the existing grayscale for components:

```
// WordPress grays.
$black: #000;			// Use only when you truly need pure black. For UI, use $gray-900.
$gray-900: #1e1e1e;
$gray-800: #2f2f2f;
$gray-700: #757575;		// Meets 4.6:1 text contrast against white.
$gray-600: #949494;		// Meets 3:1 UI or large text contrast against white.
$gray-400: #ccc;
$gray-300: #ddd;		// Used for most borders.
$gray-200: #e0e0e0;		// Used sparingly for light borders.
$gray-100: #f0f0f0;		// Used for light gray backgrounds.
$white: #fff;
```

While we can update them, we should be careful in doing so as it affects so much componentry. So I would do that separately from this PR.

Which warning does it trigger? And it's good that it does, it's another reason to merge this soon so we can address those.

Reply:
Ah, no worries, this is just a unit test that verifies that our contrast-checking algorithm is working as expected 😄

Reply:
Ah, sounds great. Thanks again!